### PR TITLE
CC-31565: Update Shipment, Service Points features

### PR DESCRIPTION
**Pull request by dmiseev**
## PR Description

https://spryker.atlassian.net/browse/CC-31565

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

**Review by abitskil — APPROVED** [review]

File: _includes/pbc/all/install-features/202311.0/install-the-service-points-feature.md [hunk 1]
```diff
@@ -114,7 +116,7 @@ Make sure that the following changes have been applied in the database:
 | spy_region.uuid                     | column | created |
 | spy_sales_order_item_service_point  | table  | created |
 
-Make sure that propel entities have been generated successfully by checking their existence. Also, make generated entity classes extending respective Spryker core classes.
+Make sure propel entities have been generated successfully by checking their existence. Also, make generated entity classes extending  respective Spryker core classes.
```
Comment (andriitserkovnyi) [inline _includes/pbc/all/install-features/202311.0/install-the-service-points-feature.md:119]: Could you elaborate on the second sentence?

File: _includes/pbc/all/install-features/202311.0/install-the-service-points-feature.md [hunk 2]
```diff
@@ -691,11 +697,11 @@ class PublisherDependencyProvider extends SprykerPublisherDependencyProvider
...
-| ServiceWritePublisherPlugin             | Listens for service events and publishes respective data. | None          | Spryker\Zed\ServicePointSearch\Communication\Plugin\Publisher\Service             |
+| ServicePointWritePublisherPlugin        | Listens for events and publishes respective data.         |           | Spryker\Zed\ServicePointSearch\Communication\Plugin\Publisher\ServicePoint        |
+| ServicePointDeletePublisherPlugin       | Listens for events and unpublishes respective data.       |           | Spryker\Zed\ServicePointSearch\Communication\Plugin\Publisher\ServicePoint        |
+| ServicePointAddressWritePublisherPlugin | Listens for events and publishes respective data.         |           | Spryker\Zed\ServicePointSearch\Communication\Plugin\Publisher\ServicePointAddress |
+| ServicePointStoreWritePublisherPlugin   | Listens for events and publishes respective data.         |           | Spryker\Zed\ServicePointSearch\Communication\Plugin\Publisher\ServicePointStore   |
```
Comment (andriitserkovnyi) [inline _includes/pbc/all/install-features/202311.0/install-the-service-points-feature.md:703]: How can they have the same spec?

File: _includes/pbc/all/install-features/202311.0/install-the-service-points-feature.md [hunk 3]
```diff
@@ -1535,8 +1624,8 @@ class ShopApplicationDependencyProvider extends SprykerShopApplicationDependency
 
 Make sure the following widgets have been registered by adding the respective code snippets to a Twig template:
 
-| WIDGET                   | VERIFICATION                                                                                                                                                 |
-|--------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| WIDGET                   | VERIFICATION                              |
+|--------------------------|-----------------------------------------------------------------|
 | ServicePointSearchWidget | `{% raw %}{%{% endraw %} widget 'ServicePointSearchWidget' args [...] only {% raw %}%}{% endraw %}{% raw %}{%{% endraw %} endwidget {% raw %}%}{% endraw %}` |
```
Comment (andriitserkovnyi) [inline _includes/pbc/all/install-features/202311.0/install-the-service-points-feature.md:1629]: What do I need to check to verify it's working?

File: _includes/pbc/all/install-features/202311.0/install-the-service-points-feature.md (hunk 2 above)
Comment (dmiseev) [reply _includes/pbc/all/install-features/202311.0/install-the-service-points-feature.md:703]: Every publisher plugin contains the same behavior, just different entities and actions (removal, creation, Address, SP, etc.) So spec could be the same, you should look at the plugin name to get the what action and what entity.

File: _includes/pbc/all/install-features/202311.0/install-the-service-points-feature.md (hunk 1 above)
Comment (dmiseev) [reply _includes/pbc/all/install-features/202311.0/install-the-service-points-feature.md:119]: I guess it should be general verification for any Integration Guide that contains new tables. So it's better to keep consistency with other IGs.

File: _includes/pbc/all/install-features/202311.0/install-the-service-points-feature.md [hunk 4]
```diff
@@ -1042,22 +1048,30 @@ class SynchronizationDependencyProvider extends SprykerSynchronizationDependency
...
 
-Make sure that when a service point is created or edited through BAPI, it is exported to Redis accordingly.
+{% endinfo_block %}
 
-Make sure that, in Redis, data is displayed in the following format:
+{% info_block warningBox "Verification" %}
+
+* Make sure that, when a service point is created or edited through BAPI, it is exported to Redis accordingly.
+
+* Make sure that, in Redis, data is displayed in the following format:
```
[the remark this answers is not in the document]
Comment (dmiseev) [reply _includes/pbc/all/install-features/202311.0/install-the-service-points-feature.md:1074]: nope, this verification belongs to Publisher plugins.

File: _includes/pbc/all/install-features/202311.0/install-the-service-points-feature.md (hunk 3 above)
Comment (dmiseev) [reply _includes/pbc/all/install-features/202311.0/install-the-service-points-feature.md:1629]: As a developer, you should add it somewhere on the twig page (for example checkout address molecule). But I didn't see previous examples where we provide such verifications for widgets.

Comment (andriitserkovnyi) [reply _includes/pbc/all/install-features/202311.0/install-the-service-points-feature.md:1629]: Yeah, but it doesn't have to be very detailed. All we are missing is a criteria to check against. After adding the widget, what should I see when I open the page with it?

File: _includes/pbc/all/install-features/202311.0/install-the-service-points-feature.md (hunk 1 above)
Comment (andriitserkovnyi) [reply _includes/pbc/all/install-features/202311.0/install-the-service-points-feature.md:119]: Do you mean I should remove the second sentence?

File: _includes/pbc/all/install-features/202311.0/install-the-service-points-feature.md (hunk 3 above)
Comment (dmiseev) [reply _includes/pbc/all/install-features/202311.0/install-the-service-points-feature.md:1629]: make sense, could you please add to the verification block: "Ensure that you can find and select a service point for your products on the `http://www.mysprykershop.com/en/checkout/address` page."

File: _includes/pbc/all/install-features/202311.0/install-the-service-points-feature.md (hunk 1 above)
Comment (dmiseev) [reply _includes/pbc/all/install-features/202311.0/install-the-service-points-feature.md:119]: nope please keep it, there are 2 possible verifications:
- We added new propel tables, columns: 
 ```
 Make sure propel entities have been generated successfully by checking their existence. Also, make generated entity classes extending respective Spryker core classes.
 ```
- We added ONLY tables:
```
Make sure propel entities have been generated successfully by checking their existence.
```

**Pull request merged**